### PR TITLE
docs(dropdown): add demo for disabled item inside menu

### PR DESCRIPTION
**Pull request by ghost**
add demo for disabled item inside menu (disabled item section)
related #3783

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 1]
```diff
@@ -82,12 +83,22 @@ export const demoComponentContent: ContentSection[] = [
...
+                      <p>Add a <code>disabled</code> class to <code>&lt;li&gt;</code>  for bootstrap 3 to make it disabled.</p>`,
+        outlet: DemoDropdownDisabledItemComponent
+      },
       {
         title: 'Menu alignment',
         anchor: 'menu-alignment',
         component: require('!!raw-loader?lang=typescript!./demos/alignment/menu-alignment.ts'),
         html: require('!!raw-loader?lang=markup!./demos/alignment/menu-alignment.html'),
         description: `<p>By default, a dropdown menu is automatically positioned 100% from the top and along
+          the left side of its parent. Add class <code>.dropdown-menu-right</code> to a <code>dropdownMenu</code>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: duplicated string

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 2]
```diff
@@ -82,6 +83,15 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/disabled-menu/disabled-menu.html'),
         outlet: DemoDropdownDisabledComponent
       },
+      {
+        title: 'Disabled item',
+        anchor: 'disabled-item',
+        component: require('!!raw-loader?lang=typescript!./demos/disabled-item/disabled-item.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/disabled-item/disabled-item.html'),
+        description: `<p>Add a <code>disabled</code> class to <code>&lt;a&gt;</code> for bootstrap 4 to make it disabled.</p>
```
Comment (valorkin) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: any reason to repeat same text twice?

File: demo/src/app/components/+dropdown/demos/disabled-item/disabled-item.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="btn-group" dropdown>
+  <button dropdownToggle type="button" class="btn btn-primary dropdown-toggle">
+    Button dropdown <span class="caret"></span>
+  </button>
+  <ul *dropdownMenu class="dropdown-menu" role="menu">
+    <li class="disabled" role="menuitem"><a class="dropdown-item disabled" href="#">Action</a></li>
```
Comment (valorkin) [inline demo/src/app/components/+dropdown/demos/disabled-item/disabled-item.html:6]: class `disabled` will not disable link itself
so you should name this demo as `mark item as disabled`
not `make it disable item`

**Review by valorkin — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+dropdown/dropdown-section.list.ts (hunk 2 above)
Comment (ghost) [reply demo/src/app/components/+dropdown/dropdown-section.list.ts]: @valorkin cause we have to add class to different tags in different bootstrap's versions

Comment (ghost) [reply demo/src/app/components/+dropdown/dropdown-section.list.ts]: joxi_screenshot_1519390484136

File: demo/src/app/components/+dropdown/demos/disabled-item/disabled-item.html (hunk 3 above)
Comment (ghost) [reply demo/src/app/components/+dropdown/demos/disabled-item/disabled-item.html:6]: @valorkin fixed it

**Review by valorkin — DISMISSED** [review]

**Pull request merged**